server/pbench/lib/config/pbench-dispatch.service.example

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Pbench Dispatch
+
+[Service]
+ExecStart=/opt/pbench-server/bin/pbench-dispatch
```

Comment:
I think we need `Environment="CONFIG=/opt/pbench-server/lib/config/pbench-server.cfg"` and similarly for the other environment variables. We also need  `User=pbench` and `Group=pbench`.